src/admin/admin.service.ts

```diff
@@ -1,97 +1,95 @@
-import { User } from '@fusionauth/typescript-client';
+import { Error, User, UserRequest, UUID } from '@fusionauth/typescript-client';
```

Comment:
Admin service should know know of FusionAuth types and should not use them. This is a `leaky` abstraction. Let's try to have the boundaries defined for this service and the FA service.

Reply:
This will be potentially bigger refactor.